web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -201,7 +201,7 @@ import {get_string_diff} from "../../src/util";
                 const replacement = after_text.substring(from, to_after);
                 // select / highlight the minimal text to be replaced
                 this.$element[0].setSelectionRange(from, to_before);
-                insert(this.$element[0], replacement);
+                insertTextIntoField(this.$element[0], replacement);
```

Comment:
(I did this yesterday in #29131.)

Reply:
Ah that's probably where the merge conflict came from, thanks!